components/Wizard/ImageStep.js

```diff
@@ -579,6 +579,8 @@ class ImageStep extends React.PureComponent {
           {/* {imageType === "vmdk" && vmwareProviderCheckbox} */}
           {requiresImageSize(imageType) && imageSizeInput}
           {(imageType === "fedora-iot-commit" || imageType === "rhel-edge-commit") && ostreeFields}
+          {imageType === "rhel-edge-container" && ostreeFields}
+          {imageType === "rhel-edge-installer" && ostreeFields}
```

Comment:
It is worth noting that this will get its own UI (see @jkozol's draft PR which introduces that), but I agree that this is the best placeholder until we land that (which we may want to wait for support in osbuild-composer first to do, as it would otherwise be untested). The generic ostree UI is a bit misleading for this image type, but if we are going to have a place holder it is better than the non-ostree one. In particular, the `parent` field does not make sense for the installer, and the help text for the `url` is not quite right (it should not refer to a parent commit). It would be fully functional though.

Reply:
I think we should leave these out. I agree with adding strings but since the `rhel-edge-installer` image-type does not accept a commit field I do not think it should be displayed. Until osbuild-composer's weldr api returns `rhel-edge-container` and `rhel-edge-installer` in the list of supported image-types these components will not display anyways. If osbuild-composer starts to support `rhel-edge-installer` before cockpit-composer has proper UI for users will be presented the commit field which isn't supported.

Reply:
> If osbuild-composer starts to support rhel-edge-installer before cockpit-composer has proper UI for it, users will be presented the commit field which isn't supported.

Yeah, it really isn't nice. The way I see it, we have two alternatives in this (hopefully very short-lived) state:
 - show a screen with an unsupported field (which can be made to work if you just ignore that field)
 -  show a screen without the required fields (which can never work at all)

I figured the former was preferable, but happy to let you call this one, there are no good solutions I think, and either way it will be short-lived / never seen.